recipes/cmip6_preprocessing/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "cmip6_preprocessing" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d34fa682b53fd353ef83c7cddddb174e26e1ecfc63cbccbac99e6157f2370750
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - pytest-runner
```

Comment:
Is this needed for install?

Reply:
I believe so. If I remove either, the local tests fail.